contracts/EpochTokenLocker.sol

```diff
@@ -184,11 +196,17 @@ contract EpochTokenLocker {
       * @param token address of ERC20 token
       * return stateIndex when withdraw was requested
       */
-    function getPendingWithdrawAmount(address user, address token)
+    function getClaimableAmount(address user, address token)
         public
         view
         returns (uint256)
     {
+        if (
+            balanceStates[user][token].pendingWithdraws.stateIndex >=
```

Comment:
Isn't this inequality the wrong way around?

Reply:
Ya, this whole thing is completely messed up. I am going to sit down with some paper and solve this properly while taking into account all possible and relevant scenarios.